### DISCUSSION TITLE
Cannot manage my apps in developer.bentley.com

### DISCUSSION THREAD

**Discussion by inigolorente** [iTwin/itwinjs-core#4746] [Q&A] [answered]
I cannot access the application management page (developer.bentley.com/apps/app-id) since last week. 

I get the following error: 
![0FAD78AC-5D47-415E-8709-414F661C15DD](https://user-images.githubusercontent.com/81552659/205161114-977e58a1-ba8d-4b81-9596-1fd6c33e854d.jpeg)

The console log shows the following:
![0727FE61-2474-4825-906C-25B837DCE8C5](https://user-images.githubusercontent.com/81552659/205161959-53435f4c-6615-4976-b938-db584d3f9fb7.png)

I raised a Service Ticket to Bentley and they suggested to post this here. Hope you can help me with this 🙏.

**Comment by aruniverse** [comment]
Just to confirm, youre trying to access [my apps](https://developer.bentley.com/my-apps/) right? The url you shared above doesnt exist.

What does it say your subscription level is in your [profile page](https://developer.bentley.com/profile/)?

**Reply by inigolorente** [reply]
Yep, i am trying to access my apps. The web shows the list of my apps, but when I click on one to manage it I get the error...

**Reply by aruniverse** [reply]
If you go to the profile page, what does it say your tier is? Does it throw an error there?
![image](https://user-images.githubusercontent.com/11051042/205164335-116bb867-fcf8-4c4e-aef8-026a3472f05a.png)

**Reply by inigolorente** [reply]
@aruniverse my profile page says that my subscription level is Trial, but I have already developed some apps within my organization that are above that tier.
Also, another colleague from my organization has the same issue.

**Reply by aruniverse** [reply]
Ive reached out internally for someone to investigate this. Sorry for getting redirected here, hope we can resolve this for you asap

**Reply by HaleRazor73** [reply]
@inigolorente, are you not seeing an "Upgrade" button when you go to the app registration under "My Apps"?  (except with the name of the subscription that your organization has, of course)

![image](https://user-images.githubusercontent.com/19479584/205171475-895487ce-e252-41a7-9ae9-a0ecd128d7f4.png)

cc @laurynasr , @irenajur

**Reply by inigolorente** [reply]
Hi @KevinHale-Bentley, I can't see that button because I can't open the App page. Before last week I had access to all of my apps, but now I get the error.

**Reply by HaleRazor73** [reply]
@inigolorente, sorry about that.  I skipped right to the actual upgrade part.  I think we'll have to ask @irenajur or one of the other developer portal developers to investigate.

**Comment by tamanud** [comment, marked as answer]
@inigolorente, can you please check if the issue is fixed on your side. Our development team have made some changes to correctly assign the proper subscription level. 
If the issue is not fixed still, please provide us with your client id for further investigation.

**Reply by inigolorente** [reply]
Hi @tamanud. It doesnt work. I get the same `Application error: a client-side exception has occurred (see the browser console for more information).` 
The error console looks the same. Are you sure the update has been pushed to live?:
![image](https://user-images.githubusercontent.com/81552659/205261375-8122e587-9c82-4ea3-873b-e65421f5a00e.png)
It's not an error for one app (i.e. https://developer.bentley.com/apps/spa/spa-aUYZ7GTjVbZFZBH3H5nktLZD4/), but in all the apps I manage.

**Reply by tamanud** [reply]
Hi @inigolorente,

From what we can see from your ClientID is that the iTwin Platform Services Subscription for your Organization(Siemens AG) have expired on 29th November 2022. 
Since there is no active Subscription form your organization you are not able to access your applications. 
You can check your organization admin once to check regarding the subscription.

**Reply by inigolorente** [reply]
Thanks @tamanud. I have asked our internal subscription manager about this.
In the meantime, I should be able to manage my applications (and getting a warning in the web instead of receiving frontend errors).
